pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/stats/BatchMetadataStoreStats.java

```diff
@@ -69,15 +75,25 @@ public BatchMetadataStoreStats(String metadataStoreName, ExecutorService executo
         EXECUTOR_QUEUE_SIZE.setChild(new Gauge.Child() {
             @Override
             public double get() {
-                return BatchMetadataStoreStats.this.executor == null ? 0 :
-                        BatchMetadataStoreStats.this.executor.getQueue().size();
+                return getQueueSize();
             }
         }, metadataStoreName);
 
         this.batchOpsWaitingChild = OPS_WAITING.labels(metadataStoreName);
         this.batchExecuteTimeChild = BATCH_EXECUTE_TIME.labels(metadataStoreName);
         this.opsPerBatchChild = OPS_PER_BATCH.labels(metadataStoreName);
 
+        var meter = openTelemetry.getMeter("org.apache.pulsar");
+        var attributes = Attributes.of(MetadataStoreStats.METADATA_STORE_NAME, metadataStoreName);
+        this.batchMetadataStoreSizeCounter = meter
+                .upDownCounterBuilder(EXECUTOR_QUEUE_SIZE_METRIC_NAME)
+                .setDescription("The number of blocking operations in the metadata store executor queue")
```

Comment:
typo here? blocking -> batch?

Reply:
I copied it over from the current description of `pulsar_batch_metadata_store_executor_queue_size` ([ref](https://pulsar.apache.org/docs/next/reference-metrics/#metadata-store-metrics)), but I think "batch" makes more sense here, thank you!